crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -332,18 +333,38 @@ async fn test_signing_error_during_genesis() {
     }
 }
 
-async fn make_signing_call(client: &mut WebsocketSender, cell: &SweetCell) -> AppResponse {
-    client
-        .request(AppRequest::ZomeCall(Box::new(ZomeCall {
-            cell_id: cell.cell_id().clone(),
-            zome_name: "sign".into(),
-            fn_name: "sign_ephemeral".into(),
-            payload: ExternIO::encode(()).unwrap(),
-            cap_secret: None,
-            provenance: cell.agent_pubkey().clone(),
-        })))
+async fn make_signing_call(
+    conductor: &SweetConductor,
+    client: &mut WebsocketSender,
+    keystore_control: &MockLairControl,
+    cell: &SweetCell,
+) -> AppResponse {
+    let reinstate_mock = keystore_control.using_mock();
+    if reinstate_mock {
+        keystore_control.use_real();
+    }
```

Comment:
@maackle yeah this was actually a pain to get right because of the way it has been used around the place

IIRC, it's kind of obvious when it is wrong though because you end up trying to use keys that aren't in the keystore you're using